#!/usr/bin/python3
def raise_exception():
    a = "Hello"
    b = 10
    c = a + b
    return c
